src/lib/support/LifetimePersistedCounter.h

```diff
@@ -87,7 +106,18 @@ class LifetimePersistedCounter : public MonotonicallyIncreasingCounter
      *
      *  @return Any error returned by a write to persistent storage.
      */
-    CHIP_ERROR PersistNextEpochStart(uint32_t aStartValue);
+    CHIP_ERROR PersistNextEpochStart(T aStartValue)
+    {
+        mNextEpoch = aStartValue;
+#if CHIP_CONFIG_PERSISTED_COUNTER_DEBUG_LOGGING
+        ChipLogDetail(EventLogging, "PersistedCounter::WriteStartValue() aStartValue 0x%x", aStartValue);
+#endif
+
+        uint32_t valueLE = Encoding::LittleEndian::HostSwap(aStartValue);
```

Comment:
Why `uint32_t`?

But also, why are we implementing this unused method instead of removing it?

Reply:
Removed